src/pages/[userId]/prospects.js

```diff
@@ -1,54 +1,157 @@
 import { useRouter } from 'next/router';
+import styled from '@emotion/styled';
 import Grid from '../../components/utils/Grid';
+import { DesktopOnly, MobileOnly } from '../../components/MediaQuery';
 
 export default function Prospects() {
   const router = useRouter();
   const { userId } = router.query;
 
+  const Container = styled.div`
+
+  h1 {
+    margin-left: 1rem;
+  }
+  .columns {
+    margin-left: -1rem;
+    padding-left: 1rem;
+  }
+
+  .name-col {
+    grid-column: 1/ span 2;
+    padding-left: 1rem;
+
+  }
+  .status-col {
+    grid-column: 3/ span 2;
+  }
+  .contact-col {
+    grid-column: 5/ span 2;
+  }
+  .notes-col {
+    grid-column: 7/ span 4;
+  }
+  .connections-col {
+    grid-column: 11/ span 2;
+  }
+
+  .dark {
+    background: #F5F5F5;
+
+  }
+
+  .numbers {
+    display: block;
+    width: 15px;
+    height: 15px;
+    text-align: center;
+    background-color: red;
+    color: white;
+    border-radius: 50%;
+    margin: 0 auto;
+  }
+
+@media (max-width: 800px){
+  font-size: .7rem;
+  text-align: center;
+
+  h1 {
+    margin-left: 0;
+  }
+
+  .name-col{
+    margin-left: -1rem;
+    grid-column: 1/ span 3;
+  }
+
+  .status-col {
+    grid-column: 4/ span 2;
+  }
+  .contact-col {
+    grid-column: 6/ span 4;
+  }
+  .connections-col {
+    grid-column: 10/ span 3;
+  }
+
+}
+  `;
+
   const prospects = [
```

Comment:
You can use this as the shape of a single prospect object for now:
```
    {
        id: 1,
        names: ['Jon Smith'],
        preferred_name: null,
        status: 'contacted',
        last_contacted: '07/13',
        notes: ['Meeting up on 08/27', 'Left a voicemail', 'Emailed'],
        others: ['Reagann Smith', 'Ryan Bristow'],
        // other fields can be dealt with later
        // don't need pledge info for now since these are prospects
    }
```

Reply:
Done